packages/ConsoleDiffer/src/config/services.yml

```diff
@@ -5,12 +5,8 @@ services:
     Rector\ConsoleDiffer\:
         resource: ..
 
-    GeckoPackages\DiffOutputBuilder\UnifiedDiffOutputBuilder:
-        arguments:
-            $options:
-                'fromFile': 'Original'
-                'toFile': 'New'
+    SebastianBergmann\Diff\Output\UnifiedDiffOutputBuilder:
 
     SebastianBergmann\Diff\Differ:
         arguments:
-            - '@GeckoPackages\DiffOutputBuilder\UnifiedDiffOutputBuilder'
+            - '@SebastianBergmann\Diff\Output\UnifiedDiffOutputBuilder'
```

Comment:
I think you want;
`'@SebastianBergmann\Diff\Output\StrictUnifiedDiffOutputBuilder'`

and keep the arguments you had for `GeckoPackages\DiffOutputBuilder\UnifiedDiffOutputBuilder` and use those for `SebastianBergmann\Diff\Output\StrictUnifiedDiffOutputBuilder`

Reply:
Thanks for help!

> I think you want;
'@SebastianBergmann\Diff\Output\StrictUnifiedDiffOutputBuilder'

What are difference between those 2?

Reply:
There are two differences between the builders:
- the `strict` one will give you more control and options (see https://github.com/sebastianbergmann/diff for all configurations options)
- `StrictUnifiedDiffOutputBuilder` will generate strict `udiff` format output that is compat for the `patch` and `git apply` commands (outputs like `diff -u` command). The non-strict `UnifiedDiffOutputBuilder` has some exceptions on the output (like additional warnings about line end differences) making it more suitable for PHPUnit.

Reply:
I see, thanks!

- https://github.com/sebastianbergmann/diff#strictunifieddiffoutputbuilder

Reply:
Actually, with my current change, everything is working :sweat_smile:
![captura de tela de 2018-02-05 13-54-05](https://user-images.githubusercontent.com/16328050/35813981-1c7f49be-0a7c-11e8-8ff4-aed23aaae161.png)

Reply:
Could you also add the previous form? There should be numbers

Reply:
yeah, you lost the "precious" line numbers ;)

Reply:
Oh, small missing, big importance.

I'll try one more time before asking @TomasVotruba help :fire:

Reply:
Here is solution: https://github.com/rectorphp/rector/pull/305#discussion_r166007142

Reply:
@TomasVotruba Fell free to finish this, I'm not handling this some some reason :cry: